server/portal/apps/onboarding/steps/mfa.py

```diff
@@ -25,15 +25,25 @@ class MFAStep(AbstractStep):
 
     def __init__(self, user):
         super(MFAStep, self).__init__(user)
+        self.user_confirm = "Confirm MFA Pairing"
 
     def display_name(self):
-        return "Confirming MFA Pairing"
+        return "Multi-Factor Authentication"
+
+    def description(self):
+        return """First setup multi-faction authentication on the
+               <a href="https://portal.tacc.utexas.edu">TACC User Portal</a>,
```

Comment:
this needs to be purple.  this should work for now but i do wonder what the best approach should be here 🤷‍♂️  (but for discussing down the road) . 
```suggestion
               <a style="color:#9d85ef;" href="https://portal.tacc.utexas.edu">TACC User Portal</a>,
```

Reply:
This is a tough one since it gets rendered as HTML sent from backend, but does not get rendered as a React component. I'll accept this change for now with the understanding that it probably won't fit compiled styles.